### PR TITLE
Cli avn service queries and metrics

### LINKED ISSUE #242: CLI docs for `avn service queries` command
# What should we add to our content collections?

(more information is useful here! Were you trying to do something particular?)

# Is this something you would work on yourself?

[ ] I plan to open a pull request for this

### PR DESCRIPTION
**Pull request by ftisiot**
# What changed, and why it matters
Added commands `service queries`, `service queries-reset` and `service metrics`

Fixes #242 #246 #237

### CODE REVIEW

File: docs/tools/cli/service.rst [hunk 1]
```diff
@@ -287,12 +287,45 @@ Service M3 commands
...
+
+::
+
+    avn service maintenance-start pg-demo
 
 ``avn service metrics``
 '''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
 
-Get service metrics
+Retrieves the metrics for a defined service.
```
[the remark this answers is not in the document]
Comment (ftisiot) [reply docs/tools/cli/service.rst:310]: Fixed

File: docs/tools/cli/service.rst [hunk 2]
```diff
@@ -287,12 +287,45 @@ Service M3 commands
...
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Parameter
+    - Information
+  * - ``service_name``
+    - The name of the service
+  * - ``--period``
```
[the remark this answers is not in the document]
Comment (ftisiot) [reply docs/tools/cli/service.rst:320]: Fixed

File: docs/tools/cli/service.rst [hunk 3]
```diff
@@ -287,12 +287,45 @@ Service M3 commands
 ``avn service maintenance-start``
 '''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
 
-Start service maintenance updates
+Starts the service maintenance updates.
```
[the remark this answers is not in the document]
Comment (ftisiot) [reply docs/tools/cli/service.rst:290]: Fixed, added a note

File: docs/tools/cli/service.rst [hunk 4]
```diff
@@ -312,12 +345,43 @@ Service Privatelink commands
 ``avn service queries``
 '''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
 
-List service query statistics
+Lists the service connections/queries statistics for an Aiven for PostgreSQL, Aiven for MySQL or Aiven for Redis service.
```
[the remark this answers is not in the document]
Comment (ftisiot) [reply docs/tools/cli/service.rst:348]: Added details on statistics

File: docs/tools/cli/service.rst [hunk 5]
```diff
@@ -312,12 +345,43 @@ Service Privatelink commands
...
+::
+  
+  avn service queries pg-demo
+
 
 ``avn service queries-reset``
 '''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
 
-Reset service query statistics
+Resets service connections/queries statistics for an Aiven for PostgreSQL, Aiven for MySQL or Aiven for Redis service.
```
[the remark this answers is not in the document]
Comment (ftisiot) [reply docs/tools/cli/service.rst:369]: Added a note about the usage

File: docs/tools/cli/service.rst [hunk 6]
```diff
@@ -287,12 +287,74 @@ Service M3 commands
...
-Start service maintenance updates
+Starts the service maintenance updates. 
+
+.. Warning::
+
+  Maintenance updates do not typically cause any noticeable impact on the service in use but may sometimes cause a short period of lower performance or downtime which shall not exceed 1 hour.
+
+
+.. Note::
+
```
Comment (lornajane) [inline docs/tools/cli/service.rst:298]: Stacking the admonitions looks a bit strange in the output. Perhaps we could move the note to after the command example?

File: docs/tools/cli/service.rst [hunk 7]
```diff
@@ -312,12 +374,52 @@ Service Privatelink commands
...
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Parameter
+    - Information
+  * - ``service_name``
+    - The name of the service
+  * - ``--fromat``
```
Comment (lornajane) [inline docs/tools/cli/service.rst:393]: Typo alert! `format` not `fromat`

File: docs/tools/cli/service.rst [hunk 8]
```diff
@@ -312,12 +374,52 @@ Service Privatelink commands
...
+    - The format string for output defining the query metrics to retrieve, e.g. `'{calls} {total_time}'`
```
Comment (lornajane) [inline docs/tools/cli/service.rst:394]: Markup weirdness here when I look at this in preview. Maybe should be double backtick to give code-formatted output that includes the single quotes?

**Review by lornajane — APPROVED** [review]
Couple of _really_ minor things here - could fix and then merge? I'm approving :)

File: docs/tools/cli/service.rst (hunk 6 above)
Comment (ftisiot) [reply docs/tools/cli/service.rst:298]: Fixed!

File: docs/tools/cli/service.rst (hunk 7 above)
Comment (ftisiot) [reply docs/tools/cli/service.rst:393]: Fixed

File: docs/tools/cli/service.rst (hunk 8 above)
Comment (ftisiot) [reply docs/tools/cli/service.rst:394]: yep done

**Pull request merged**